qcodes/instrument_drivers/Keysight/keysightb1500/KeysightB1500_module.py

```diff
@@ -300,3 +317,17 @@ def clear_timer_count(self) -> None:
         (FMT3 and FMT4).
         """
         self.root_instrument.clear_timer_count(chnum=self.channels)
+
+
+class StatusMixin:
```

Comment:
doesn't mypy require "param1"/"param2"/"names" attributes to be defined? in init or somehwere else? if so, good.

Reply:
I have changed the implementation. This one look a bit convoluted to me.